src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/__snapshots__/DataGuidanceFileWriterTests.WriteFile_FileWithEmptyProperties.snap

```diff
@@ -1,10 +0,0 @@
-﻿Test publication
```

Comment:
Wow, good spot, looks like these snapshot files have been hanging around since a change back in August 2021!

Reply:
Haha yeh! This is a potential problem with snapshot tests if you change test method names, something to bear in mind in the future.